### ISSUE TITLE
Accessibility privilege given to "Terminal" (OS X)

### ISSUE THREAD

**Issue by sughodke** [ActivityWatch/activitywatch#78]
On the first run, a pop-up displays saying I need to authorize the app for Accessibility use.  However, OS X only gives me the option to authorize the Terminal, and not specifically ActivityWatch.

This could be a security concern as it opens un-necessary permissions for other Terminal apps.  Ideally ActivityWatch should be packaged as an App (?), and that way Accessibility privileges can be granularly handed out.

ActivityWatch 0.7.0b1
MacOS Sierra 10.12.5 (16F73)

<img width="780" alt="screen shot 2017-06-22 at 3 59 07 pm" src="https://user-images.githubusercontent.com/31588/27459568-c075b962-5763-11e7-85e4-73cd68917ea8.png">

**Comment by ErikBjare** [comment]
You are right! This was planned but I realize now that I never created an issue for it. 

This will hopefully be landing in one of the later betas.

**Comment by ErikBjare** [comment]
Looks unlikely to make it into 0.7 right now, sorry.

**Comment by jwiese** [comment]
@sughodke You're successfully running on Sierra 10.12.5? When running (release version, not from source) aw-qt, I get the following message:
```
Error loading Python lib '/Applications/activitywatch/.Python': dlopen(/Applications/activitywatch/.Python, 10): image not found
```
While I'm not certain, this seems like it might be related to the macOS System Integrity Protection (SIP)? If I'm doing something wrong then this is just a documentation issue, otherwise it seems like there is some other bug.

**Comment by ErikBjare** [comment]
@jwiese Thanks for testing out macOS support, it used to work but I haven't personally run it on macOS for a long time. It's been my plan to test it, but that plan fell through due to other priorities.

 - Does running aw-server, aw-watcher-afk and aw-watcher-window independently work fine?
 - If not, running from source is very likely to still work since it's tested in CI.

I will definitely look into this, but I need to get my hands on a Mac and I can't promise when that will happen. Right now the best bet on macOS is to install from source.

**Comment by jwiese** [comment]
Just to close the loop on this, I tested this. None of them worked initially, but they worked from source. aw-qt still does not work (at least, not yet), but aw-server, aw-watcher-afk, and aw-watcher-window work fine independently, as does aw-webui.

Thanks!

**Comment by ErikBjare** [comment]
I've finally managed to borrow a MacBook Air which I'll do some testing on soon. Will make an attempt to create a `.app` file but not sure how the packaging would work given the multiple processes.

Use of the PyInstaller builds on macOS and Linux is discouraged until issues like this one are resolved: https://github.com/ActivityWatch/activitywatch/issues/110

**Comment by ErikBjare** [comment]
I've started work towards creating an app bundle, but had some issues: https://github.com/ActivityWatch/aw-qt/pull/28

**Comment by jwiese** [comment]
Not sure if this is still useful, but this was the output I was getting after building from source:
```
$ aw-qt
Traceback (most recent call last):
  File "/Users/jasonwiese/anaconda/bin/aw-qt", line 11, in <module>
    load_entry_point('aw-qt==0.1', 'console_scripts', 'aw-qt')()
  File "/Users/jasonwiese/anaconda/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 565, in load_entry_point
  File "/Users/jasonwiese/anaconda/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 2598, in load_entry_point
  File "/Users/jasonwiese/anaconda/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 2258, in load
  File "/Users/jasonwiese/anaconda/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 2264, in resolve
  File "/Users/jasonwiese/anaconda/lib/python3.6/site-packages/aw_qt-0.1-py3.6.egg/aw_qt/__init__.py", line 1, in <module>
    from . import resources
ImportError: cannot import name 'resources'
```

Might just be something dumb on my end

**Comment by ErikBjare** [comment]
That is actually really helpful! Nothing dumb on your end :)

I know what's the cause of that. Thanks for the stacktrace :+1: 

The file missing should be created by [this command in the aw-qt Makefile](https://github.com/ActivityWatch/aw-qt/blob/52b28375ee3834e743846e68919b5eb7bd503091/Makefile#L11). You could try running it manually and see what went wrong. I'm a bit surprised that the command didn't cause `make build` to fail so curious what happens if you run it manually.

**Comment by ErikBjare** [comment]
@jwiese The `Error loading Python lib '/Applications/activitywatch/.Python': dlopen(/Applications/activitywatch/.Python, 10): image not found` bug was fixed in #119. The just released v0.7.0b4 release contains the fix 😃

**Comment by ErikBjare** [comment]
This was fixed a long time ago, and especially so in the `.app` bundle that will be available in the next release.
